[rospy.client][INFO] 2021-12-14 10:05:58,969: init_node, name[/server_endpoint], pid[13293]
[xmlrpc][INFO] 2021-12-14 10:05:58,969: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-12-14 10:05:58,970: Started XML-RPC server [http://wng:39987/]
[rospy.init][INFO] 2021-12-14 10:05:58,970: ROS Slave URI: [http://wng:39987/]
[rospy.impl.masterslave][INFO] 2021-12-14 10:05:58,970: _ready: http://wng:39987/
[rospy.registration][INFO] 2021-12-14 10:05:58,970: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-12-14 10:05:58,970: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-12-14 10:05:59,070: registered with master
[rospy.rosout][INFO] 2021-12-14 10:05:59,070: initializing /rosout core topic
[rospy.rosout][INFO] 2021-12-14 10:05:59,072: connected to core topic /rosout
[rospy.simtime][INFO] 2021-12-14 10:05:59,072: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-12-14 10:05:59,078: Starting server on 192.168.0.8:10000
[rospy.internal][INFO] 2021-12-14 10:05:59,305: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-12-14 10:06:07,877: Connection from 192.168.0.8
[rosout][INFO] 2021-12-14 10:06:07,882: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-12-14 10:06:07,952: topic[/execution_check] adding connection to [http://wng:39987/], count 0
[rospy.internal][INFO] 2021-12-14 10:06:07,954: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rosout][ERROR] 2021-12-14 10:09:17,936: Exception: No more data available
[rosout][INFO] 2021-12-14 10:09:17,938: Disconnected from 192.168.0.8
[rospy.core][INFO] 2021-12-14 10:09:21,559: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-12-14 10:09:21,567: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][INFO] 2021-12-14 10:09:21,567: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-12-14 10:09:21,568: topic[/execution_check] removing connection to http://wng:39987/
[rospy.impl.masterslave][INFO] 2021-12-14 10:09:21,568: signal-2
[rospy.internal][WARNING] 2021-12-14 10:09:21,569: Unknown error initiating TCP/IP socket to wng:35345 (http://wng:39987/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 359, in read_ros_handshake_header
    raise ROSHandshakeException("connection from sender terminated before handshake header received. %s bytes were received. Please check sender for additional details."%b.tell())
ROSHandshakeException: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.

[rospy.core][INFO] 2021-12-14 10:09:21,670: signal_shutdown [atexit]
